docs/Logger.md

```diff
@@ -114,45 +113,61 @@ char str_sensor2[10];
 
 /****************************************
  * Main Functions
- ****************************************/ 
+ ****************************************/
 void setup() {
   Serial.begin(115200);
   Serial1.begin(115200);
 }
 
 void loop() {
 
-  /* Analog reading */ 
-  float sensor1 = analogRead(A0);
-  float sensor2 = analogRead(A1);
-  
-  /* 4 is mininum width, 2 is precision; float value is copied onto str_sensor*/
-  dtostrf(sensor1, 4, 2, str_sensor1);
-  dtostrf(sensor2, 4, 2, str_sensor2);
-
-  /* Building the logger command */ 
-  sprintf(command, "init#");
-  sprintf(command, "%s%s/%s|%s|%s|", command, USER_AGENT, VERSION, METHOD, TOKEN);
-  sprintf(command, "%s%s:%s=>", command, DEVICE_NAME, DEVICE_LABEL);
-  sprintf(command, "%s%s:%s", command, VARIABLE_LABEL_1, str_sensor1);
-  //sprintf(command, "%s,%s:%s", command, VARIABLE_LABEL_2, str_sensor2); // uncomment this line to send sensor 2 values
-  sprintf(command, "%s|end#final", command);
-
-  /* Prints the command sent */
-  //Serial.println(command);// uncomment this line to print the command
-  
-  /* Sends the command to the telemetry unit */
-  Serial1.print(command);
+  /* Wait for the server response to read the values and built the command */
+  if (flag) {
```

Comment:
please add a comment that explains the main purpose of this flag. Also, a more detailed variable name should be used.